### ISSUE TITLE
ORCID Record page to mention Arctos as well

### ISSUE THREAD

**Issue by KyndallH** [ArctosDB/arctos#2155]
From Beth Wommack @Jegelewicz : I was wondering if there was something we should also do on our ORCID Record page to mention Arctos as well?

[https://arctosdb.org/orcids/]

**Comment by Jegelewicz** [comment]
Well, you could add your agent page url to the websites section of your ORCID profile.

![image](https://user-images.githubusercontent.com/5725767/61008128-00463380-a32c-11e9-938a-96977efd2ed8.png)

![image](https://user-images.githubusercontent.com/5725767/61008217-3388c280-a32c-11e9-9042-f1a72f618776.png)

**Comment by ewommack** [comment]
Thanks Teresa!
Do you think that would be helpful? Since you mention that ORCID is the new way that aggregators figure out agent names, I wasn't sure if there was a good way for us to provide additional data for non-Arctos viewers of the ORCID page.

**Comment by Jegelewicz** [comment]
Having your ID in Arctos should do the trick eventually. As GBIF and others work through this use case, we will also need to get those IDs to pass to them when we pass our data. It won't happen overnight and we need to get a bunch in so that it can all be tested.

**Comment by ewommack** [comment]
It will be nice when that happens. I can't wait to be able to really track different agents through museums like we do in Arctos.
I'll try adding the Arctos Agent Page to the ORCID ID, and maybe that will help someone else along the line.
Thanks!

**Comment by ewommack** [comment]
Since Agent Pages are only accessible to those with access to Arctos, is adding the link to the Agent page going to be helpful only to Arctos users?
![Screen Shot 2019-07-10 at 4 16 32 PM](https://user-images.githubusercontent.com/18562884/61008939-456b6500-a32e-11e9-839a-3eb4475e3f0b.png)

**Comment by Jegelewicz** [comment]
> I can't wait to be able to really track different agents through museums like we do in Arctos.

Check out [BloodHound](https://bloodhound-tracker.net/)

Here is @mkoo [Bloodhound page](https://bloodhound-tracker.net/0000-0003-0410-722X)
Note the "[deposited at](https://bloodhound-tracker.net/0000-0003-0410-722X/deposited-at)" tab.

Once you have an ORCID, the next step is to create your Bloodhound page....

**Comment by Jegelewicz** [comment]
> Since Agent Pages are only accessible to those with access to Arctos, is adding the link to the Agent page going to be helpful only to Arctos users?

Agent pages are publicly accessible. Log out and select Search>Agents
https://www.dropbox.com/s/7ox6st3a8unmf9x/Screenshot%202019-07-10%2016.22.56.png?dl=0

Or log out and try to access [my page](http://arctos.database.museum/agent.cfm?agent_id=21300608&agent_name=Teresa%20J.%20Mayfield-Meyer).

**Comment by dustymc** [comment]
The Search>Agents page should work for everyone.

>agent page url to the websites section of your ORCID profile.

Nice!

https://github.com/ArctosDB/arctos/issues/2141 would be one mechanism by which we might force-feed ORCID (and other agent info) to the world. We could probably stuff ORCIDs in the EMLs as well.

>through museums

Yes, that's the key - maybe stuff like this will push others towards the idea that collectors-and-such are more than just strings in cells.

ORCIDs+DOIs+FundRef can also do some fairly wild things - funding produces specimens which get cited in publications which get referenced in more publications, etc.; a dead rat ultimately and mostly-indirectly can support a LOT of research. That relies on authors submitting ORCIDs (I think I've come across one publication with them...), using fundref (not sure I've even seen that from "us"), and citing specimens in such a way that machines can understand (which seldom happens). Maybe this is a step down that path.

**Comment by Jegelewicz** [comment]
You also might want to see: https://github.com/gbif/portal16/issues/297

**Comment by ewommack** [comment]
> Agent pages are publicly accessible. Log out and select Search>Agents
https://www.dropbox.com/s/7ox6st3a8unmf9x/Screenshot%202019-07-10%2016.22.56.png?dl=0

Ah I see what I did wrong. I pasted in the page when I was logged in and editing the Agent info and not the page that is accessed through the search function for Agents. I've got it fixed now.

> Check out BloodHound
And that's another thing to work on. Always new things to learn! Thanks!
